src/popup/queryClient.ts

```diff
@@ -0,0 +1,27 @@
+import { createAsyncStoragePersister } from '@tanstack/query-async-storage-persister';
+import { QueryClient } from '@tanstack/react-query';
+import { storage } from 'webextension-polyfill';
+
+export const queryClient = new QueryClient({
+	defaultOptions: {
+		queries: {
+			gcTime: 1000 * 60 * 60 * 24, // 24 hours
```

Comment:
My main concern about caching is about getting updated provider connections from our APIs; if a user connects an integration and goes straight to the popup, I'd expect they would be able to grant permissions right away.

Reply:
The only hook that uses the provider library right actually already has a stale time set to 10 minutes (with immediate invalidation if the user clicks on one of the links, since there's a reasonable chance that the user may perform an action on the PR).

Reply:
I can still add a global 10 second stale time though